examples/ai-lab-example/tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
     "outDir": "dist",
-    "rootDir": "."
+    "rootDir": ".",
+    "paths": {
+      "ai-lab": ["../../packages/ai-lab/src/index.ts"],
```

Comment:
This fixes an issue for me where the types were not working in the example. If this has been working for others I can drop it from this PR.

Reply:
I was getting the same type error ever since it was removed. Thanks for bringing it back!